content/en/usage/setup.md

```diff
@@ -1,7 +1,99 @@
+
+You can install Node.js, NPM, and the tweasel CLI on Ubuntu 23.04[^ubuntu-node] like this:
+
+[^ubuntu-node]: Note: The repositories of older Ubuntu versions contain versions of Node.js that are too old. On these, you have to install Node.js in another way, for example via [nvm](https://github.com/nvm-sh/nvm) or the [NodeSource packages](https://github.com/nodesource/distributions).
+
+```sh
+sudo apt update
+sudo apt install nodejs npm
+
+sudo npm i -g tweasel-cli
+```
+
+The libraries, on the other hand, should be installed locally in a project (i.e. without `-g`), for example for cyanoacrylate:
+
+```sh
+npm i cyanoacrylate
+```
+
+### Setup for physical devices
+
+If you want to work with physical devices, some manual setup is necessary, depending on the platform.
+
+The setup for Android is described in the [Android developer documentation](https://developer.android.com/studio/run/device#setting-up). On Ubuntu, the user must be a member of the `plugdev` group (`sudo usermod -aG plugdev <username>`) and `udev` rules for the device must be installed (`sudo apt install android-sdk-platform-tools-common`). Tips for other Linux distributions can be found at [`android-udev-rules`](https://github.com/M0Rf30/android-udev-rules).
+
+For iOS, additional setup steps are only necessary under Windows. Here, you have to install the Apple Device Driver and the Apple Application Support. You can get those by installing iTunes.
+
+## Device preparation
+
+Our libraries work with physical phones and emulators on Android and physical phones on iOS. We test with the following target (but other versions may also work):
+
+| Platform | Type | Tested versions |
+| --- | --- | --- |
+| Android | `device` (Moto G7 Power) | 13 (API level 33) |
+| Android | `emulator` | 11 (API level 30), 13 (API level 33) |
+| iOS | `device` (iPhone X, iPhone 6S) | 15.6.1, 16.0 |
+
+Depending on what kind of device you want to use for the analysis, the preparation steps described below are necessary. Installing and setting up all other dependencies on the device is done automatically by our tools.
+
+### Physical Android phones
+
+On physical Android phones, USB debugging must be enabled. This can be done via *Settings* -> *System* -> *Developer options* (to activate, tap *Build number* seven times under *Settings* -> *About phone* -> *Android version*) -> *USB debugging*.
+
+To be able to do traffic analysis in a meaningful way, the device needs to be rooted. For this, we recommend [Magisk](https://topjohnwu.github.io/Magisk/). To use that, you also have to unlock the bootloader. The instructions for that vary from device to device. Usually, all data on the device is lost in the process.  
+After rooting, root debugging should be enabled if available: *Settings* -> *System* -> *Developer options* -> *Rooted debugging*.
```

Comment:
Do you have to anything other than accept the prompt when it comes up (I still don't see how it would be possible to do that beforehand as you mentioned before)?